pennylane_lightning/core/src/simulators/lightning_kokkos/bindings/LKokkosBindings.hpp

```diff
@@ -143,7 +189,9 @@ void registerBackendClassSpecificBindings(PyClass &pyclass) {
         .def("collapse", &StateVectorT::collapse,
              "Collapse the statevector onto the 0 or 1 branch of a given wire.")
         .def("normalize", &StateVectorT::normalize,
-             "Normalize the statevector to norm 1.");
+             "Normalize the statevector to norm 1.")
```

Comment:
Is this `normal` used in the python layer?

Reply:
I have now removed the normalize here.